packages/embark/src/cmd/cmd_controller.js

```diff
@@ -164,6 +164,9 @@ simulator(_options) {
         engine.registerModuleGroup("storage");
         engine.registerModuleGroup("cockpit");
         engine.registerModulePackage('embark-deploy-tracker', {plugins: engine.plugins});
+        
+        // load custom plugins
+        engine.config.plugins.loadPlugins();
```

Comment:
Do you need to add this to other commands too?

Reply:
I'm not sure actually, that's why I held off adding them.